gita/info.py

```diff
@@ -113,8 +142,8 @@ def get_info_items() -> List[str]:
     return display_items
 
 
-def get_path(prop: Dict[str, str]) -> str:
-    return f'{Color.cyan}{prop["path"]}{Color.end}'
+def get_path(prop: Dict[str, str], truncator: Truncate) -> str:
```

Comment:
Here we pass in the Truncate class and the call the truncate member function to justify the text